core/includes/form.inc

```diff
@@ -4749,7 +4751,10 @@ function theme_textarea($variables) {
  */
 function theme_password($variables) {
   $element = $variables['element'];
-  $element['#attributes']['type'] = 'password';
+  $element['#attributes'] = array(
+    'type' => 'password',
+    'spellcheck' => 'false',
+  );
```

Comment:
The problem with this change is that you are "wiping out" all the other key/values of the `$element['#attributes]` array  (including "disabled"). This is why the test is failing.

Instead do this:
```php
$element['#attributes']['type'] = 'password'l
$element['#attributes']['spellcheck'] = 'false';
```

or

```php
  $element['#attributes'] += array(
    'type' => 'password',
    'spellcheck' => 'false',
  );
```

The same thing applies to all other changes in this PR.

Reply:
(Notice the `+=`)

Reply:
Thanks. Done.